src/utils/file_cache.ml

```diff
@@ -85,7 +87,7 @@ struct
         true
       end
       else begin
-        false
+        false (* TODO: should dispose + remove ? *)
```

Comment:
I agree that we can `dispose` and `remove`, but was it a question?
Should we do that here? I think the function `check` is not supposed to update the Hashtbl, but I might be wrong.

Reply:
Right, it would also be possible to just remove the TODO :) Otherwise I think it's fine for `check` to clean up the hashtbl once it discovers expired elements, since they won't ever be useful.